homeassistant/runner.py

```diff
@@ -73,16 +73,6 @@ def new_event_loop(self) -> asyncio.AbstractEventLoop:
         loop.set_default_executor = warn_use(  # type: ignore
             loop.set_default_executor, "sets default executor on the event loop"
         )
-
-        # Shut down executor when we shut down loop
-        orig_close = loop.close
```

Comment:
close already shuts down the executor.  At least it appears to have done so since cpy3.5+